README.md

```diff
@@ -55,8 +55,8 @@ Run 'bspguy <command> help' to read about a specific command.
 ### Windows users:
 1. Install CMake and Visual Studio
 1. Download and extract the source somewhere
-1. Download [Dear ImGui](https://github.com/ocornut/imgui/releases/tag/v1.76) and extract next to the `src` folder. Rename to `imgui`.
-1. Download [GLFW](https://www.glfw.org/) and extract next to the `src` folder. Rename to `glfw`.
+1. Download [Dear ImGui](https://github.com/ocornut/imgui/releases/latest/) and extract next to the `bspguy` folder. Rename to `imgui`.
+1. Download [GLFW](https://www.glfw.org/) and extract next to the `bspguy` folder. Rename to `glfw`.
```

Comment:
"next to" means to extract beside, so that the folders are at the same level. Extracting next to the `bspguy` folder would mean extracting a level higher than it should be.

I get that the wording is confusing. Usually people say to extract _into_ some folder, not _next to_ it. I didn't want to say "into the bspguy folder" because it might not be called "bspguy" depending on how you cloned the repo.

Reply:
But "Open a command prompt in the `bspguy` folder" used in text ) I think folder always bspguy))

Reply:
Okay

Reply:
I guess the repo would always be called "bspguy-master" if you click the download button, which is what I assume windows users would do. In that case you can say "into the bspguy-master folder"